### ISSUE TITLE
k in kmaxavg

### ISSUE THREAD

**Issue by aminvenv** [terrier-org/pyterrier#275]
kmaxavg aggregation method in “pyterrier.text" uses 2 as the k parameter. 

```
if self.agg == 'kmaxavg':
    rtr = topics_and_res.groupby(['qid', 'olddocno'])['score'].apply(lambda ser: ser.nlargest(2).mean()).reset_index().rename(columns={'olddocno' : 'docno'})
    from .model import query_columns
    #add query columns back
    rtr = rtr.merge(topics_and_res[query_columns(topics_and_res)].drop_duplicates(), on='qid')
```

**Comment by cmacdonald** [comment]
Thank you @aminvenv, good spot. Probably this class also needs a unit test, which would have picked this up.

**Comment by cmacdonald** [comment]
Fix and unit test committed. This will be included in the next 0.8.x release. Thanks for the report @aminvenv
